aiocache/backends/memory.py

```diff
@@ -10,32 +10,32 @@ class SimpleMemoryBackend:
     Wrapper around dict operations to use it as a cache backend
     """
 
-    _cache: Dict[str, object] = {}
-    _handlers: Dict[str, asyncio.TimerHandle] = {}
-
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
+        self._cache: Dict[str, Any] = {}
```

Comment:
```suggestion
        self._cache: Dict[str, object] = {}
```

I think our internal code should be safe regardless of the value.

Reply:
Not sure what safety means here. I haven't seen `objeft` used this way anywhere since `Any` type is spicifically made for cases where you don't care about type

Reply:
No, `Any` is used to disable type checking because you have no way to guarantee type safety. `object` is type safe, meaning that the code is expected to work with anything passed to it.

For example, `print()` works with anything: https://github.com/python/typeshed/blob/main/stdlib/builtins.pyi#L1511
It is always type safe.

Reply:
An improvement that we could look at later, would be to make these classes `Generic`, then we can make the value `_T` and have the methods `.get()` etc. return `_T`. But, that's something for another PR.